crates/sb_core/js/bootstrap.js

```diff
@@ -244,18 +248,25 @@ globalThis.bootstrapSBEdge = (opts, isUserRuntime, isEventManager) => {
     args: readOnly([]), // args are set to be empty
     mainModule: getterOnly(() => ops.op_main_module()),
   });
+  ObjectDefineProperty(globalThis, "Deno", readOnly(denoOverrides));
 
   setNumCpus(1); // explicitly setting no of CPUs to 1 (since we don't allow workers)
   setUserAgent("Supabase Edge Runtime");
   setLanguage("en");
 
-  ObjectDefineProperty(globalThis, "Deno", readOnly(denoOverrides));
 
-  if(isUserRuntime) {
-    loadUserRuntime();
+  if(isUserWorker) {
+    delete globalThis.EdgeRuntime;
```

Comment:
@andreespirela I've inlined the `loadUserRuntime` here. It's handy to have it access to the scope here (so it can access `core` and `ops`.)